.travis.yml

```diff
@@ -12,6 +12,8 @@ install:
   - pip install coveralls
 script:
   - flake8
+  - python manage.py makemessages
+  - python manage.py compilemessages
```

Comment:
we shouldn't add these in travis file, the command makemessages should always be done locally, also we already compile the messages in the [docker file ](https://github.com/praekelt/molo-gem/blob/develop/Dockerfile#L7)